tests/unit/test_timestream.py

```diff
@@ -9,7 +9,7 @@
 
 import awswrangler as wr
 
-from .._utils import is_ray_modin
+from .._utils import get_time_str_with_random_suffix, is_ray_modin
 
```

Comment:
Lol these are big tests just to check that a log gets printed haha

Reply:
Tbh I could live without tests for logging :-)